cid/helpers/quicksight/__init__.py

```diff
@@ -906,6 +897,33 @@ def describe_dataset(self, id, timeout: int=1) -> Dataset:
 
         return self._datasets.get(id, None)
 
+    def get_dataset_last_ingestion(self, dataset_id) -> str:
+        """returns human friendly status of the latest ingestion"""
+        try:
+            ingestions = self.client.list_ingestions(
+                DataSetId=dataset_id,
+                AwsAccountId=self.account_id,
+            ).get('Ingestions', [])
+        except self.client.exceptions.ResourceNotFoundException:
+            return '<RED>NotFound<END>'
+        except self.client.exceptions.AccessDeniedException:
+            return '<YELLOW>AccessDenied<END>'
+        if not ingestions:
+            return None
+        last_ingestion = ingestions[0] # Suppose it is the latest
+        status = last_ingestion.get('IngestionStatus')
+        time_ago = ago(last_ingestion.get('CreatedTime'))
+        if last_ingestion.get('ErrorInfo', {}).get('Type') == "DATA_SET_NOT_SPICE":
+            return '<BLUE>DIRECT_QUERY<END>'
```

Comment:
why not also define status here, and return it at line 925  return f'{status} {time_ago}'  ? it's the same login as in UI time ago is the creationt ime.

Reply:
I mean nothing ciriminal if statys liek this only  helps with code consistenncy

Reply:
If dataset is DATA_SET_NOT_SPICE the status can be empty